[2021-07-02 05:37:48,729] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-04-08T05:55:00+00:00 [queued]>
[2021-07-02 05:37:48,773] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2020-04-08T05:55:00+00:00 [queued]>
[2021-07-02 05:37:48,773] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:37:48,773] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-07-02 05:37:48,774] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:37:48,822] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): print_date> on 2020-04-08T05:55:00+00:00
[2021-07-02 05:37:48,831] {standard_task_runner.py:52} INFO - Started process 143141 to run task
[2021-07-02 05:37:48,848] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'print_date', '2020-04-08T05:55:00+00:00', '--job-id', '209', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpjuux4svy', '--error-file', '/tmp/tmpqwxmul38']
[2021-07-02 05:37:48,851] {standard_task_runner.py:77} INFO - Job 209: Subtask print_date
[2021-07-02 05:37:48,978] {logging_mixin.py:104} INFO - Running <TaskInstance: test.print_date 2020-04-08T05:55:00+00:00 [running]> on host 9b3ed935c8bc
[2021-07-02 05:37:49,136] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@airflow.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2020-04-08T05:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-08T05:55:00+00:00
[2021-07-02 05:37:49,138] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-02 05:37:49,139] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'date']
[2021-07-02 05:37:49,163] {subprocess.py:75} INFO - Output:
[2021-07-02 05:37:49,167] {subprocess.py:79} INFO - Fri Jul  2 05:37:49 UTC 2021
[2021-07-02 05:37:49,168] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-02 05:37:49,297] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test, task_id=print_date, execution_date=20200408T055500, start_date=20210702T053748, end_date=20210702T053749
[2021-07-02 05:37:49,523] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-02 05:37:49,578] {local_task_job.py:151} INFO - Task exited with return code 0
